### ISSUE TITLE
Can I use ORSSerialPort send TTL  serial communication

### ISSUE THREAD

**Issue by pwjpj52** [armadsen/ORSSerialPort#149]
My board only support TTL  communication, I CAN use usb to ttl connect to my mac, so Can I use ORSSerialPort send TTL  serial communication?
thank u very much
Looking forward to your reply

**Comment by armadsen** [comment]
Yes. Most if not all USB to serial adapters have TTL output levels anyway, so what you’re describing is commonplace.

**Comment by pwjpj52** [comment]
> Yes. Most if not all USB to serial adapters have TTL output levels anyway, so what you’re describing is commonplace.
> […](#)
> On Nov 18, 2019, at 11:22 PM, pwjpj52 ***@***.***> wrote: ﻿ My board only support TTL communication, I CAN use usb to ttl connect to my mac, so Can I use ORSSerialPort send TTL serial communication? thank u very much Looking forward to your reply — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Thank you very much, you have helped me a lot.
